scripts/format

```diff
@@ -30,8 +30,16 @@ find . \
     \( -type f -name '*.h' -or -name '*.cc' -or -name '*.proto' \) \
   \) -exec clang-format -i -style=file {} +
 
+# Format the core `oak` workspace.
 cargo fmt --all
 
+# Format the `oak_abi` and `oak_utils` crates.
+cargo fmt --all --manifest-path=oak_abi/Cargo.toml
+cargo fmt --all --manifest-path=oak_utils/Cargo.toml
```

Comment:
Why only these crates need to be formatted?
Should `oak_runtime` and `oak_loader` be formatted too?